Documentation/getting-started.md

```diff
@@ -0,0 +1,193 @@
+
+```yaml mdox-exec="cat example/user-guides/getting-started/prometheus.yaml"
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: prometheus
+spec:
+  serviceAccountName: prometheus
+  resources:
+    requests:
+      memory: 400Mi
+```
+
+To verify that the instance is up and running, run:
+
+```bash
+kubectl get -n default prometheus prometheus -w
+```
+
+For more information, see the [Prometheus Operator RBAC guide]({{< ref "rbac" >}}).
+
+## Deploying Alertmanager
+
+Let us take a simple example that creates 3 replicas of Alertmanager.
+
+```yaml mdox-exec="cat example/user-guides/alerting/alertmanager-example.yaml"
+apiVersion: monitoring.coreos.com/v1
+kind: Alertmanager
+metadata:
+  name: example
+spec:
+  replicas: 3
+```
+
+Wait for all Alertmanager pods to be ready:
+
+```bash
+kubectl get pods -l alertmanager=example -w
+```
+
+However, Alertmanager as it is now is of no use to us. To properly use Alertmanager, it is important to understand the relationship between Prometheus and Alertmanager. Alertmanager is used to:
+
+* Deduplicate alerts received from Prometheus.
+* Silence alerts.
+* Route and send grouped notifications to various integrations (PagerDuty, OpsGenie, mail, chat, …).
+
+So, to put Alertmanager instances to use, you would need to integrate it with Prometheus.
+
+## Integrating Alertmanager With Prometheus
+
+### Exposing the Alertmanager service
+
+To access the Alertmanager interface, you have to expose the service to the outside. For
+simplicity, we use a `NodePort` Service.
+
+```yaml mdox-exec="cat example/user-guides/alerting/alertmanager-example-service.yaml"
+apiVersion: v1
+kind: Service
+metadata:
+  name: alertmanager-example
+spec:
+  type: NodePort
+  ports:
+  - name: web
+    nodePort: 30903
+    port: 9093
+    protocol: TCP
+    targetPort: web
+  selector:
+    alertmanager: example
+```
+
+Once the Service is created, the Alertmanager web server is available under the
+node's IP address on port `30903`.
+
+> Note: Exposing the Alertmanager web server this way may not be an applicable solution. Read more about the possible options in the [Ingress guide](user-guides/exposing-prometheus-and-alertmanager.md).
+
+### Configuring Alertmanager in Prometheus
+
+The Alertmanager cluster is now fully functional and highly available, but no
+alerts are fired against it.
+
+First, create a Prometheus instance that will send alerts to the Alertmanger cluster:
+
+```yaml mdox-exec="cat example/user-guides/alerting/prometheus-example.yaml"
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: example
+spec:
+  serviceAccountName: prometheus
+  replicas: 2
+  alerting:
+    alertmanagers:
+    - namespace: default
+      name: alertmanager-example
+      port: web
+  serviceMonitorSelector:
+    matchLabels:
+      team: frontend
+  ruleSelector:
+    matchLabels:
+      role: alert-rules
+      prometheus: example
```

Comment:
This part doesn't seem related to the topic "Connecting Prometheus to Alertmanagers". I wonder if this could be considered extra cognitive load 🤔 

We can use a YAML code block here without the mdox-exec part that way we're not limited to always have an example file somewhere.